qa/run_sc_tests.sh

```diff
@@ -0,0 +1,196 @@
+#!/bin/bash
+set -e -o pipefail
+
+#Log stdout to file, use '| tee /dev/fd/3' to also log to console
+#Must match logfile location referenced in sc_test_framework
+exec 3>&1 1>>"${BASH_SOURCE%/*}/sc_test.log" 2>&1
+
+#Check Environment Variables have been set
+if [[ -z "$BITCOINCLI" ]]; then
+    echo "Environment Variable: BITCOINCLI not set" | tee /dev/fd/3
+    exit 1
+fi
+if [[ -z "$BITCOIND" ]]; then
+    echo "Environment Variable: BITCOIND not set" | tee /dev/fd/3
+    exit 1
+fi
+if [[ -z "$SIDECHAIN_SDK" ]]; then
+    echo "Environment Variable: SIDECHAIN_SDK not set" | tee /dev/fd/3
+    exit 1
+fi
+
+# parse args
+for i in "$@"; do
+  case "${i}" in
+    -extended)
+      EXTENDED="true"
+      shift
+      ;;
+    -exclude=*)
+      EXCLUDE="${i#*=}"
+      shift
+      ;;
+    -split=*)
+      SPLIT="${i#*=}"
+      shift
+      ;;
+    *)
+      # unknown option/passOn
+      passOn+="${i} "
+      ;;
+  esac
+done
+
+#Run the tests
+testScripts=(
```

Comment:
What do you think about moving all the test scripts into a subfolder so we can iterate over all the files inside it instead of listing them all manually?

Reply:
I agree with this Paolo, perhaps this can be changed in a future PR